cni/pkg/nodeagent/server.go

```diff
@@ -273,7 +273,7 @@ func (s *meshDataplane) AddPodToMesh(ctx context.Context, pod *corev1.Pod, podIP
 		// regardless of ztunnel's current status.
 		// So annotate indicating that this pod was injected (and thus needs to be either retried
 		// or uninjected on removal) but is not fully captured yet.
-		log.Errorf("failed to add pod to ztunnel: %v, pod partially added, annotating with pending status")
+		log.Errorf("failed to add pod to ztunnel: pod partially added, annotating with pending status")
```

Comment:
The messages which have no formatting should probably use log.Error or similar

Reply:
```suggestion
		log.Error("failed to add pod to ztunnel: pod partially added, annotating with pending status")
```